source/documentation/deploying_services/redis.md

```diff
@@ -193,34 +195,34 @@ Type `yes` when asked for confirmation.
 
```

Comment:
Beyond the scope of the PR: Why is data classification under 'Maintaining the service'?

Reply:
Agree